src/service/diagnosticReports.service.ts

```diff
@@ -0,0 +1,36 @@
+import { SweepArguments } from '../interfaces/generaltask';
+import { MetaArguments } from '../interfaces/mySolutions';
+import { pluralCompare } from './generalTask.service';
+
+export const filterMetaArguments = (metaArguments: ReadonlyMeta, sweepArguments: string[]): MetaArguments => {
+  const filteredMeta: MetaArguments = [];
+  metaArguments.map((kv) => {
```

Comment:
Filter wasn't working I think because the graphql type was readonly, and I couldn't iterate them for rendering. Error said something like 'type has no call expressions'. This was to change to type to not `readonly`. Is this right?

Reply:
I see, thanks for clarifying!